pkg/k8shandler/visualization.go

```diff
@@ -177,19 +163,33 @@ func (clusterRequest *ClusterLoggingRequest) removeKibana() (err error) {
 	if clusterRequest.isManaged() {
 		name := "kibana"
 		proxyName := "kibana-proxy"
-		if err = clusterRequest.RemoveDeployment(name); err != nil {
+		if err = clusterRequest.removeKibanaCR(); err != nil {
 			return
 		}
 
-		if err = clusterRequest.RemoveOAuthClient(proxyName); err != nil {
+		if err = clusterRequest.RemoveSecret(name); err != nil {
 			return
 		}
 
-		if err = clusterRequest.RemoveSecret(name); err != nil {
+		if err = clusterRequest.RemoveSecret(proxyName); err != nil {
 			return
 		}
 
-		if err = clusterRequest.RemoveSecret(proxyName); err != nil {
+	}
+
+	return nil
+}
+
+func (clusterRequest *ClusterLoggingRequest) removeOldKibana() (err error) {
```

Comment:
add documentation around this function -- why we're doing this and that we'll be removing it after a release

Reply:
shouldn't this function also be removing the old service, route, etc?

Reply:
It is removing everything related to the CLO's kibana, like deployment, service, configmap, route, etc.

Reply:
oh ack, i didn't expand the source :laughing: